### ISSUE TITLE
Bug: Empty Schedule

### ISSUE THREAD

**Issue by ChrisLaganiere** [uclaradio/uclaradio-iOS#11]
Some of our users are reporting an empty schedule, where once the data loads for the schedule, nothing is shown in the schedule view controller.

<img width="400" alt="screen shot 2017-02-07 at 9 21 27 pm" src="https://cloud.githubusercontent.com/assets/2042745/23007472/ba4014a2-f3bd-11e6-9f0c-ee4933b7021c.png">

I investigated the issue on an affected device and found that the `Schedule` data model sent to `ScheduleViewController` was empty, with no shows for each day.. The json data is returned successfully from the server, but fails to get converted to data models because `Show.showFromJSON` is failing at line 68:

```
if let components = DateFormatter().formatShowTimeStringToDateComponents(day + " " + time) {
```

So the schedule sent to scheduleVC is empty:

<img width="199" alt="screen shot 2017-02-15 at 8 35 04 pm" src="https://cloud.githubusercontent.com/assets/2042745/23007557/58cbbdba-f3be-11e6-8b7b-4b580cdba434.png">

**Comment by ChrisLaganiere** [comment]
So we likely need to fix `DateFormatter+ShowDate: formatShowTimeStringToDateComponents`

**Comment by nathunsmitty** [comment]
That makes sense. Weird that it's only happening on some devices, though... If it's an issue with parsing json from the api, I'd think we'd see this on every device because they're all making the same request. Any chance we can read the raw json sent to an affected phone?

**Comment by ChrisLaganiere** [comment]
yeah... I'll try to get that

**Comment by ChrisLaganiere** [comment]
Fixed in #13
